### PR TITLE
Lic 268/dropdown example

### PR DESCRIPTION
**Pull request by rohanrjohn**

### CODE REVIEW

File: src/CustomStyleCommand.ts [hunk 1]
```diff
@@ -547,26 +554,49 @@ export function compareMarkWithStyle(
   if (style) {
     switch (mark.type.name) {
       case MARKSTRONG:
-        same = undefined !== style[STRONG];
+        if (undefined === mark.attrs[ATTR_OVERRIDDEN] || (undefined !== mark.attrs[ATTR_OVERRIDDEN] && !mark.attrs[ATTR_OVERRIDDEN])) {
```
Comment (Tezra) [inline src/CustomStyleCommand.ts:557]: contents of this if should be a function to reduce duplication and clarify what the condition is.

File: src/customStyle.ts [hunk 2]
```diff
@@ -5,11 +5,10 @@ import {
   RESERVED_STYLE_NONE_NUMBERING,
 } from './CustomStyleNodeSpec.js';
 import { DEFAULT_NORMAL_STYLE } from './Constants.js';
-import { EditorView } from 'prosemirror-view';
 let customStyles = new Array(0);
 let styleRuntime;
 let hideNumbering = false;
-let _view: EditorView;
+let _view;
```
Comment (Tezra) [inline src/customStyle.ts:11]: why is the type removed? No any.

File: src/customStyle.ts [hunk 3]
```diff
@@ -23,6 +22,19 @@ function isValidStyleName(styleName) {
   );
 }
 
+export function addStyleToList(style) {
```
Comment (Tezra) [inline src/customStyle.ts:25]: no any

File: src/customStyle.ts [hunk 4]
```diff
@@ -59,7 +71,7 @@ export function getCustomStyleByName(name: string): Style {
   return style;
 }
 
-export function setView(csview: EditorView) {
+export function setView(csview) {
```
Comment (Tezra) [inline src/customStyle.ts:74]: no any

File: src/index.ts [hunk 5]
```diff
@@ -267,8 +288,19 @@ export function onUpdateAppendTransaction(
...
     }
+    tr = tr?.scrollIntoView();
+  }
+
+  return tr;
+}
+
+
+//LIC-254 Create new line by placing cursor at the beginning of a paragraph applies the current style instead of Normal style
+export function applyStyleForPreviousEmptyParagraph(nextState, tr) {
```
Comment (Tezra) [inline src/index.ts:299]: no any

File: src/ui/CustomStyleEditor.tsx [hunk 6]
```diff
@@ -25,7 +26,7 @@ import { AlertInfo } from './AlertInfo.js';
 
 let customStyles: Style[] = [];
 const otherStyleSelected = false;
-const editedStyles = [];
+let editedStyles = [];
```
Comment (Tezra) [inline src/ui/CustomStyleEditor.tsx:29]: Why is this being changed if it is only being set to `[]` in the constructor?

**Review by Tezra — CHANGES_REQUESTED** [review]

**Review by rodgersgb — APPROVED** [review]

**Pull request merged**